Source/JavaScriptCore/jit/PCToCodeOriginMap.cpp

```diff
@@ -220,8 +223,10 @@ PCToCodeOriginMap::PCToCodeOriginMap(PCToCodeOriginMapBuilder&& builder, LinkBuf
     m_pcRangeEnd = linkBuffer.locationOf<NoPtrTag>(builder.m_codeRanges.last().end).dataLocation<uintptr_t>();
     m_pcRangeEnd -= 1;
 
+    m_OMGTailCallInlinedOriginFlags.resize(builder.m_codeRanges.size());
```

Comment:
I think m_OMGTailCallInlinedOriginFlags should be sparse

Reply:
Switched this to a vector holding the codeRange indices. Since we expect this to be a tiny vector, I used `contains` for the lookup in `findPC`.